pages/get-involved.html

```diff
@@ -52,7 +52,7 @@ <h5><a class="headline-link" href="#djs">{% trans "DJs" %}</a></h5>
       {% trans "There are many ways to help out as a DJ:" %}
       <ul>
         <li>
-          {% blocktrans with controller_wiki_link="https://github.com/mixxxdj/mixxx/wiki/start#controller_mapping_documentation" %}<a href="{{ controller_wiki_link }}">Create or improve mappings</a> for your controller.{% endblocktrans %}
+          {% blocktrans with controller_wiki_link="https://github.com/mixxxdj/mixxx/wiki#controller_mapping_documentation" %}<a href="{{ controller_wiki_link }}">Create or improve mappings</a> for your controller.{% endblocktrans %}
```

Comment:
```suggestion
          {% blocktrans with controller_wiki_link="https://github.com/mixxxdj/mixxx/wiki#controller-mapping-documentation" %}<a href="{{ controller_wiki_link }}">Create or improve mappings</a> for your controller.{% endblocktrans %}
```

otherwise the anchor doesnt work